lib/rubocop/cop/mixin/allowed_methods.rb

```diff
@@ -3,7 +3,9 @@
 module RuboCop
   module Cop
     # This module encapsulates the ability to allow certain methods when
-    # parsing.
+    # parsing. Function to allow even code that would be an offense in this module.
```

Comment:
Function to allow even code that would be an offense in this module. - this sentence confuses me. 

Perhaps we'd be better off without it. You can also see the comment I left for the ticket that spawned this conversation.

Reply:
Thank you for your comments. I have improved the expression. How is it now?